cypress/e2e/cloud/usage.test.ts

```diff
@@ -108,16 +109,14 @@ describe('Usage Page PAYG With Data', () => {
           }).then(() => {
             cy.visit(`/orgs/${id}/usage`)
             cy.getByTestID('usage-page--header').should('be.visible')
+            cy.setFeatureFlags({quartzIdentity: true, multiOrg: true})
           })
         })
       })
     )
   })
 
   it('should display the usage page with data for a PAYG user', () => {
-    // The implication here is that there is no Upgrade Now button
-    cy.get('.cf-page-header--fluid').children().should('have.length', 1)
```

Comment:
This check is now useless. It's true that this page-header will have length one, but that's because the upgrade now button is moved into the header.

The better way to test this is by checking that the upgrade button _actually_ isn't present, rather than implying it's not present based on the size of the header.